lib/rules/selector-anb-no-unmatchable/README.md

```diff
@@ -0,0 +1,75 @@
+# selector-anb-no-unmatchable
+
+Disallow unmatchable An+B selectors.
+
+<!-- prettier-ignore -->
+```css
+a:nth-child(0n+0) {}
+/*↑             ↑
+ * This unmatchable An+B selector */
+```
+
+[An+B selectors](https://drafts.csswg.org/css-syntax/#anb) are one-indexed. Selectors that always evaluate to `0` will not match any elements.
```

Comment:
question

Should it link to a draft?
Would https://www.w3.org/TR/css-syntax-3/#anb-microsyntax be better?